commands/connect.js

```diff
@@ -1,24 +1,35 @@
-const chalk = require('chalk');
-const files = require('../lib/files');
-const git = require('../lib/git');
-const getConfig = require('../lib/config').getConfig;
+const chalk = require("chalk");
+const files = require("../lib/files");
+const git = require("../lib/git");
+const getConfig = require("../lib/config").getConfig;
 
-const connect = (username, project, connection, opts) => {
-  console.log(chalk.cyanBright('Connecting remote origin...'));
-  const githubUsername = username || getConfig('github');
+const connect = (username, project, connection, organization, opts) => {
+  console.log(chalk.cyanBright("Connecting remote origin..."));
+  const githubUsername = username || getConfig("github");
   const projectName = project || files.getCurrentDirectoryBase();
-  const connectionType = connection || getConfig('connection');
+  const connectionType = connection || getConfig("connection");
```

Comment:
but what happens if `getConfig` fails? What if the config just hasn't occurred? does `getConfig` throw an error, or does it just pass (which is bad)?

Reply:
do this after you integrate typescript

Reply:
Moved this out of this file it will log errors before this function is called